### PR TITLE
Update Issue Templates

### PR DESCRIPTION
**Pull request by beorostica**
This pull request is intended to resolve issue https://github.com/cncastillo/KomaMRI.jl/issues/228

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
 name: Feature Request
-description: Suggest a new feature
+description: Suggest a new julia feature
 title: "[FR]: "
-labels: ["feature"]
+labels: ["feature", "enhancement"]
```
Comment (cncastillo) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:4]:
```suggestion
labels: ["enhancement"]
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -1,14 +1,13 @@
 name: Feature Request
-description: Suggest a new feature
+description: Suggest a new julia feature
 title: "[FR]: "
```
[the remark this answers is not in the document]
Comment (cncastillo) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:3]: Remove the title suggestion for the bug report as well.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -13,24 +13,33 @@ body:
...
       required: true
   - type: textarea
     id: environment
     attributes:
       label: Environment
-      description: What environment are you using? Run the following code to get all the relevant information ```using Pkg; println("OS $(Base.Sys.MACHINE)"); println("Julia $VERSION"); [println("$(pkg.name) $(pkg.version)") for (_, pkg) in Pkg.dependencies() if occursin("KomaMRI", pkg.name)];```
-      placeholder: Output of `using Pkg; println("OS $(Base.Sys.MACHINE)"); println("Julia $VERSION"); [println("$(pkg.name) $(pkg.version)") for (_, pkg) in Pkg.dependencies() if occursin("KomaMRI", pkg.name)];`
-      render: Julia
+      description: |
+        What environment are you using? Run the following code to get all the relevant information:
```
Comment (cncastillo) [inline .github/ISSUE_TEMPLATE/bug.yaml:23]:
```suggestion
        We need a few details to understand the issue better. Please run the following code to easily get all the relevant information (works for Julia >=1.9):
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (beorostica) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/257/commits/ee252284a2bcad050cadcd955a006a3779bb56de

**Pull request merged**